docs/encryption.md

```diff
@@ -0,0 +1,45 @@
+# Configure Image Encryption
+This document describes the method to configure image encryption for `containerd` for use with the `cri` plugin.
+
+
+## Encrypted Container Images
+
+Encrypted container images are OCI images which contain encrypted blobs. An example of how these encrypted images can be created through the use of [containerd/imgcrypt project](https://github.com/containerd/imgcrypt). In order for the containerd runtime to be able to decrypt these images, the `cri` has to pass the correct options in its calls to containerd. This includes material such as keys and encryption metadata required by the runtime.
+
+## Key Models
+
+
+Encryption ties trust to an entity based on the model in which a key is associated with it. We call this the key model. One such usecase is when we want to tie the trust of a key to the node in a cluster. In this case, we call it the "node" Key Model. Future work will include more key models to facilitate other trust associations (i.e. for multi-tenancy).
+
+
+1. "node" Key Model - In this model encryption is tied to workers. The usecase here revolves around the idea that an image should be only decryptable only on trusted host. Although the granularity of access is more relaxed (per node), it is beneficial because there various node based technologies which help bootstrap trust in worker nodes and perform secure key distribution (i.e. TPM, host attestation, secure/measured boot). In this scenario, runtimes are capable of fetching the necessary decryption keys. An example of this is using the [`--decryption-keys-path` flag in imgcrypt](https://github.com/containerd/imgcrypt).
+
+
+## Configuring image encryption "node" key model
+
+The default configuration does not handle encrypted image. 
+
+In order to set up image encryption, create/modify `/etc/containerd/config.toml` as follows:
```

Comment:
s/encryption/encryption handling... an encrypted handler... or just go with decryption...  an example for configuring the node key model:...